examples/react-tsx/src/js/app.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
```

Comment:
You could use `import { render } from "react-dom";` to just import the render funktion

Reply:
thx. done 79d87ad